### PR TITLE
Depreciation messages 

### PR DESCRIPTION
**Pull request by feedforward**
#397 Tensorflow 1.8 depreciation messages handled.

### CODE REVIEW

**Comment by feedforward** [comment]
@npapernot , Please have a look at this, handles the left over warnings from #437 .

File: cleverhans/attacks.py [hunk 1]
```diff
@@ -50,6 +51,12 @@ def __init__(self, model, back='tf', sess=None, dtypestr='float32'):
         # We are going to keep track of old graphs and cache them.
         self.graphs = {}
 
+        # Depriciation warning
```
Comment (goodfeli) [inline cleverhans/attacks.py]: Fix spelling, should be "Deprecation"

**Comment by goodfeli** [comment]
Once the spelling error is fixed, this is OK to merge.

**Comment by npapernot** [comment]
My bad @feedforward, I suggested earlier to replace the deprecated call inline for a tutorial script but had not realized that there were that many changes needed throughout the CleverHans module as well. If that many lines need to be changed, it would be better to create a compat module indeed

**Comment by feedforward** [comment]
@npapernot that is not an issue, i can roll back the changes to my previous PR.

so as per @goodfeli 's suggestion, will create compact.py

**Comment by goodfeli** [comment]
Thanks. Is this ready to re-review?

**Comment by feedforward** [comment]
@goodfeli , Travis test are running as of now, should be done 30 minutes.
Please review it after this.

**Comment by goodfeli** [comment]
Also, so clean and pretty! Thanks for the refactor

**Comment by feedforward** [comment]
@goodfeli Thanks ...

**Pull request merged**